app/models/patient.rb

```diff
@@ -156,7 +159,10 @@ class Patient < ApplicationRecord
   before_update :set_time_zone, if: proc { |patient|
     patient.monitored_address_state_changed? || patient.address_state_changed?
   }
-  before_create :set_time_zone
+  before_create do
+    set_time_zone
+    enrolled_isolation ? 'Isolation' : 'Exposure'
```

Comment:
I don't think this line is doing anything is it? I'm thinking we could accomplish what we want with something like:
```ruby
    self.enrolled_isolation = isolation
```